src/lib.rs

```diff
@@ -0,0 +1,153 @@
+mod abis;
+mod pb;
+mod rpc;
+
+use pb::token_tracker::{AirTransfers, AirTransfer, AirErc1155TransferBatch, AirErc1155TransferSingle};
+use substreams::store;
+use substreams::{log, Hex, proto};
+use substreams::errors::Error;
+use substreams_ethereum::{pb::eth::v2 as eth, Event as EventTrait};
+
+#[substreams::handlers::map]
+fn map_transfers(blk: eth::Block) -> Result<AirTransfers, Error> {
+    let mut air_transfers = vec![];
+    let mut air_transfer_batchs = vec![];
+    let mut air_transfer_singles = vec![];
+    for log in blk.logs(){
+        if let Some(event) = abis::erc20::events::Transfer::match_and_decode(log){
+            log::info!("transfer event: {}", Hex(&event.to));
+            // log::info!("ERC20 token: {}, {}, {}", Hex(log.log.clone().address), log.receipt.transaction.hash.clone().to_string(), log.ordinal());
+
+            let air_transfer: AirTransfer = AirTransfer{
+                amount: event.value.to_string(),
+                token_id: Hex(log.log.clone().address).to_string(),
```

Comment:
Doesnt seem correct.

Reply:
Which line exactly?

Reply:
I have changed token_id to token_address for this